test/conformance/kubeconform/link.go

```diff
@@ -45,23 +45,30 @@ func link(o *options) {
 			return nil
 		})
 	if err != nil {
-		fmt.Printf("%v", err)
-		return
+		return err
+	}
+	fmt.Println()
+	fmt.Printf("Using behaviors from these %d files:\n", len(behaviorFiles))
+	for _, f := range behaviorFiles {
+		fmt.Println("    ", f)
```

Comment:
Is the space just for readability?

Reply:
yes. though if primary consumer ends up being machines we'll want to tweak this output